torch/onnx/_internal/fx/fx_exporter.py

```diff
@@ -129,6 +129,58 @@ def apply(
         return tuple(arg for arg in model_args if arg is not None), {}
 
 
+class RemoveNonTensorInputStep:
+    """Remove the non-tensor input arguments.
+
+    Dynamo does not support non-tensor input arguments (https://github.com/pytorch/pytorch/issues/99534).
```

Comment:
Please also mention that the concrete value of the non-tensor input is ~~burnt~~ embedded into the graph.

As well as the suggestion from Meta to rewrite the model code to make it tensor.

We might need to further investigate the feasibility of that suggestion. This altogether would make a great diagnostic, with auto fix-it attached!

Reply:
Sure!